am-lib/src/main/java/uk/gov/hmcts/reform/amlib/AccessManagementService.java

```diff
@@ -112,7 +112,8 @@ public void grantExplicitResourceAccess(@NotNull @Valid ExplicitAccessGrant acce
      */
     @AuditLog("explicit access revoked by '{{mdc:caller}}' to resource '{{accessMetadata.resourceId}}' "
         + "defined as '{{accessMetadata.serviceName}}|{{accessMetadata.resourceType}}|{{accessMetadata.resourceName}}' "
-        + "from accessor '{{accessMetadata.accessorId}}': {{accessMetadata.attribute}}")
+        + "from accessor '{{accessMetadata.accessorId}}': {{accessMetadata.attribute}} "
```

Comment:
I think better would be `from accessor {{accessMetadata.accessorId}} with relationship {{accessMetadata.relationship}}: {{accessMetadata.attribute}}`

This way we keep information that describes user together.

Reply:
👍